autoemulate/experimental/emulators/base.py

```diff
@@ -190,7 +192,12 @@ class PyTorchBackend(nn.Module, Emulator, Preprocessor):
     verbose: bool = False
     preprocessor: Preprocessor | None = None
     loss_fn: nn.Module = nn.MSELoss()
+    optimizer_cls: type[optim.Optimizer] = optim.Adam
     optimizer: optim.Optimizer
+    lr: float = 1e-1
+    scheduler_cls: type[LRScheduler] | partial[LRScheduler] = ExponentialLR
+    scheduler: LRScheduler | None = None
+    gamma: float = 0.9
```

Comment:
I think we could do something like this everywhere
```suggestion
        scheduler_cls: type[LRScheduler] | partial[LRScheduler] | None = None
```